main.py

```diff
@@ -87,13 +89,23 @@ async def get_experiment_info(file: str) -> Any:
 
 
 @app.get("/experiment/submit/")
-async def submit_experiment(file: str, args: str = "{}") -> int:
+async def submit_experiment(
+    file: str,
+    args: str = "{}",
+    pipeline: str = "main",
+    priority: int = 0,
+    timed: Optional[str] = None
+) -> int:
     """Submits the given experiment file.
     
     Args:
         file: The path of the experiment file.
         args: The arguments to submit which must be a JSON string of a dictionary.
           Each key is an argument name and its value is the value of the argument.
+        pipeline: The pipeline to run the experiment in.
+        priority: The priority that higher value means sooner scheduling.
+        timed: The due date for the experiment of which format is "YYYY-MM-DDTHH:MM:SS".
+          If there is no due date, it is set to None.
```

Comment:
Make description in the perspective of users' view. For example:
"None for no due date."